snowflake/udfs/udfs/traffic_drop/traffic_drop_udf_test.py

```diff
@@ -0,0 +1,177 @@
+import unittest
+import random
+
+from traffic_drop_udf import *
+    
+class TestTrafficDrop(unittest.TestCase):
+    detectionID = "88d98d56-a3a4-43ea-b4f8-8a231077c6a9"
+
+    flows_aggregated = [
+        [
+            'antrea-test#"Pod-A"',
```

Comment:
why the double quotes around the Pod name (but not the namespace)? I don't understand why when I look at the SQL query for running the UDTF.

Reply:
Thanks pointing out! I created a bad test case here, there should be no quotes around pod name.